parasol/prior/nnds.py

```diff
@@ -24,9 +24,12 @@ def forward(self, q_Xt, q_At):
         XAt = T.concatenate([Xt, At], -1)
         p_Xt1 = self.network(XAt)
         if isinstance(p_Xt1, stats.Gaussian):
+            sigma = T.reshape(p_Xt1.get_parameters('regular')[0], T.concatenate([leading_dim, [ds, ds]]))
+            delta_mu = T.reshape(p_Xt1.get_parameters('regular')[1], T.concatenate([leading_dim, [ds]]))
+            mu = np.add(q_Xt.get_parameters('regular')[1], delta_mu)
```

Comment:
I think this needs to be a regular add (`q_Xt.get_parameters('regular')[1] + delta_mu`) not an `np.add`.